src/components/api-reference/scoring.js

```diff
@@ -1037,6 +1106,44 @@ export default class ScoringEngine extends PureComponent {
         <p className='ds-text--lead'>
           Last, the Score Object is passed back to the QPP Submissions API, which builds the application response by inserting the Score Object into the response body and returns this response to the requester. This response body contains JSON describing in detail the record of the current aggregate estimate of the submission score.
         </p>
+        <h1 className='ds-h1'>Score Object Navigation</h1>
+        <p className='ds-text--lead'>
+          To help facilitate finding scoring details in the Scoring Output data structure, example navigation to important scoring details is outlined. A sample submission with corresponding scoring output is provided for reference.
+        </p>
+        <div>
+          <h2 className='ds-h2'>Example Submission</h2>
```

Comment:
Is it helpful to put the example submission here? It's a lot of information to scroll through and it seems more important to present how to navigate the score object rather than demonstrate the relationship between score and submission.

Reply:
We have a ticket -- somewhere, can't find it -- to break this down into sub-pages per your original request when we first created this page. We don't have it prioritized at this time, but it will be prioritized in the future to break this content into a more readable format.

Reply:
👍